.config/nextest.toml

```diff
@@ -23,6 +26,10 @@ fail-fast = false
 
 test-threads = 2
 
+[[profile.default.overrides]]
```

Comment:
Slightly counterintuitive this. It should slow down the tests but it actually speeds them up quite a bit. When I run the tests with `cargo nextest run` and without this config, they take about 1:20 to run on my laptop. Running them the same way with this config, they take about 0:24... So it's just a case of the wasm compiler and a few other things probably, going incredibly slowly when 7 tests are launched at once. On CI I was seeing tests time out waiting for the app to be ready. Things seem much better running these in serial.

Reply:
Hellz yeah!